pkg/signal/trap_linux_test.go

```diff
@@ -0,0 +1,80 @@
+// +build linux
+
+package signal
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"syscall"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestTrap(t *testing.T) {
+	sigmap := map[string]os.Signal{
+		"TERM":          syscall.SIGTERM,
+		"QUIT":          syscall.SIGQUIT,
+		"INT":           os.Interrupt,
+		"TERM_MULTIPLE": syscall.SIGTERM,
+		"INT_MULTIPLE":  os.Interrupt,
+	}
+	tmpDir, err := ioutil.TempDir("/tmp", "main")
+	defer os.Remove(tmpDir)
```

Comment:
I think the first arg (`/tmp`) should be the empty string ("") so that it uses the normal temp directory lookup, instead of forcing it to `/tmp`.